packages/tables/src/Concerns/CanSearchRecords.php

```diff
@@ -108,7 +108,7 @@ protected function applyColumnSearchesToTableQuery(Builder $query): Builder
      */
     protected function extractTableSearchWords(string $search): array
     {
-        return explode(' ', preg_replace('/\s+/', ' ', $search));
+        return $search === '' ? [''] : str_getcsv(preg_replace('/\s+/', ' ', $search), ' ');
     }
```

Comment:
`str_getcsv()` returns `[null]` if `$search` is empty string, so it was wrapped so if empty it returns `['']` bypassing the need to run `str_getcsv()`

Reply:
Is there a reason we should ever return `['']`? IMO, if there are no search terms, we should return `[]`

Maybe we need to filter the array before returning, and only allow terms that are `filled()`

Reply:
> Is there a reason we should ever return `['']`? IMO, if there are no search terms, we should return `[]`
> 
> Maybe we need to filter the array before returning, and only allow terms that are `filled()`

Not sure to be honest, I didn't have the time to dive in deep enough to see if it could ever be passed an empty string. Regardless however, if this method gets re-used elsewhere, would they know to ensure it is not passed an empty string. I guess if we want to be safe, we could have it throw an exception if it was passed an empty string.

Reply:
I've refactored it to filter the array of empty strings